rpc/codec/server.go

```diff
@@ -79,6 +87,49 @@ func (c *serverCodec) ReadRequestHeader(r *rpc.Request) error {
 	return nil
 }
 
+// UserRequest is an interface for RPC requests that have a "requested user".
+type UserRequest interface {
```

Comment:
This has moved to security/tls.go, inside the authentication method and unexported.